rails_event_store_active_record/Makefile

```diff
@@ -14,18 +14,18 @@ install: ## Install gem dependencies
 
 test: ## Run unit tests
 	@echo "Running unit tests"
-	@bundle exec rspec
+	@BUNDLE_GEMFILE=Gemfile bundle exec rspec
```

Comment:
Was it necessary after all?

Reply:
yes because otherwise it remembers (couldn't find where on filesystem...) the last one from

```
 install: ## Install gem dependencies
 	@echo "Installing gem dependencies"
 	@bundle install
 	@echo "Installing v1_v2_schema_migration gem dependencies"
 	@BUNDLE_GEMFILE=spec/v1_v2_schema_migration/Gemfile bundle install
 	@BUNDLE_GEMFILE=spec/without_rails/Gemfile bundle install
```